<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>638</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="vl_nodeeditor">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewProject"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit_Item"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionNodeWindow"/>
    <addaction name="actionPropertyWindow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dw_left">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>139</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>524287</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dwc_left">
    <property name="minimumSize">
     <size>
      <width>240</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>240</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="vl_treewidget" stretch="0,0">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="le_tree_filter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AICCTreeWidget" name="tw_node">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragOnly</enum>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dw_right">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>230</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>524287</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dwc_right">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>245</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>245</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Property</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dw_toolbar">
   <property name="minimumSize">
    <size>
     <width>416</width>
     <height>139</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tw_toolbar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>110</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>110</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(240, 240, 240)</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_simulation">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <widget class="QToolButton" name="tb_new">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/res/New_24.png</normaloff>:/res/New_24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>48</y>
           <width>40</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>New
▼</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_open">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>10</y>
           <width>71</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/res/Open.png</normaloff>:/res/Open.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_save">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>71</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/res/Save.png</normaloff>:/res/Save.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_print">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>71</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Print</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/res/Print.png</normaloff>:/res/Print.png</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>90</width>
           <height>90</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align:bottom;
background-repeat:repeat-no-repeat;
background-image:url(:/res/LibraryBrower.png);
background-position:top;
background-origin:content;
padding-top:8px;
padding-bottom:15px;</string>
         </property>
         <property name="text">
          <string>
Labrary
Browser</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="Line" name="line">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>3</width>
           <height>70</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="Line" name="line_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>3</width>
           <height>70</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_ros_network">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>10</y>
           <width>90</width>
           <height>90</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align:bottom;
background-repeat:repeat-no-repeat;
background-image:url(:/res/configureRosNetwork_24.png);
background-position:top;
background-origin:content;
padding-top:8px;
padding-bottom:15px;</string>
         </property>
         <property name="text">
          <string>
ROS
Network</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>10</y>
           <width>90</width>
           <height>90</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align:bottom;
background-repeat:repeat-no-repeat;
background-image:url(:/res/rosVariableSizeMessages_24.png);
background-position:top;
background-origin:content;
padding-top:8px;
padding-bottom:15px;</string>
         </property>
         <property name="text">
          <string>
VariableSize
Messages</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>10</y>
           <width>90</width>
           <height>90</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align:bottom;
background-repeat:repeat-no-repeat;
background-image:url(:/res/Config.png);
background-position:top;
background-origin:content;
padding-top:8px;
padding-bottom:15px;</string>
         </property>
         <property name="text">
          <string>Model
Settings</string>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="Line" name="line_3">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>20</y>
           <width>3</width>
           <height>70</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Page</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewProject">
   <property name="text">
    <string>Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEdit_Item">
   <property name="text">
    <string>Edit Item</string>
   </property>
  </action>
  <action name="actionNodeWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NodeWindow</string>
   </property>
  </action>
  <action name="actionPropertyWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PropertyWindow</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AICCTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">aicctreewidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
